lib/Util/Notices/EmailWithInvalidListNotice.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace MailPoet\Util\Notices;
+
+use MailPoet\Cron\Workers\SendingQueue\SendingQueue;
+use MailPoet\WP\Functions as WPFunctions;
+use MailPoet\WP\Notice;
+
+class EmailWithInvalidListNotice {
+  const OPTION_NAME = SendingQueue::EMAIL_WITH_INVALID_LIST_OPTION;
```

Comment:
Nothing wrong here, I am just curious. Why do you copy the const from Sending Queue instead of using in the code the const from the queue?

Reply:
I noticed that all Notice classes uses OPTION_NAME and actually it is used in [the notices controller for dismissing](https://github.com/mailpoet/mailpoet/blob/master/lib/Util/Notices/PermanentNotices.php#L87-L107). So I wanted to keep it similar.